spec/features/movies/show_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'rails_helper'
+
+RSpec.describe 'movie show page' do
+  before(:each) do
+    @user1 = User.create!(name: 'Ana', email: 'ana@example.com')
+    @movie1 = Movie.new(title: 'The Godfather', id: 238, genres:[
+        {
+            "id": 18,
```

Comment:
I wonder if we need the id in this scenario, could we git away with doing just the `name` ?